### PR TITLE
[Programmes 6885] podcast structured data

### PR DESCRIPTION
**Pull request by tomwinnington**
Ticket:
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6885

Link:
http://sandbox.bbc.co.uk/programmes/b006qpgr/episodes/downloads

Gives the following:
screen shot 2019-01-18 at 09 50 10

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/Podcast/PodcastController.php [hunk 1]
```diff
@@ -87,4 +89,31 @@ public function __invoke(
             'genre' => $genre,
         ]);
     }
+
+    protected function getSchemaForProgrammeContainerAndParents(StructuredDataHelper $structuredDataHelper, Programme $programmeContainer): array
```
Comment (BjornTwachtmann) [inline src/Controller/Podcast/PodcastController.php]: This looks identical to what we already have in BaseProgrammeEpisodesController. Is that right? If so, could we move this whole function into the StructuredDataHelper and call it from there, avoiding the duplication?

**Comment by tomwinnington** [comment]
Test this please

**Comment by Tasiobg** [comment]
I had a look again and the collection changes looks good to me

**Pull request merged**